reflex/github.json

```diff
@@ -1,6 +1,6 @@
 {
   "owner": "reflex-frp",
   "repo": "reflex",
-  "rev": "240932e52185d196263f01da38511f6e05c0d959",
-  "sha256": "03nr3nscs9cjns2nv45pyyipkhsbwhdwimyl009a80sgfd63mnah"
+  "rev": "95e0744c86c9e233cd4050b89196ec65fb7b4e0a",
```

Comment:
Point to `develop` once https://github.com/reflex-frp/reflex/pull/241 is merged

Reply:
merged

Reply:
thanks!